libraries/core/src/main/kotlin/com/bumble/appyx/core/composable/Child.kt

```diff
@@ -38,8 +38,8 @@ fun <NavTarget : Any, State> ParentNode<NavTarget>.Child(
     transitionParams: TransitionParams,
     transitionHandler: TransitionHandler<NavTarget, State>,
     decorator: @Composable ChildTransitionScope<State>.(
-        child: ChildRenderer,
-        transitionDescriptor: TransitionDescriptor<NavTarget, State>
+        ChildRenderer,
+        TransitionDescriptor<NavTarget, State>
```

Comment:
This is no longer allowed in Kotlin 2.0
https://developer.android.com/jetpack/androidx/releases/compose-compiler#1.5.0

`Named arguments for @Composable lambda calls have been deprecated. This feature is relying on internal compiler APIs and will not be supported by K2.`